manuscript/chapter1.md

```diff
@@ -520,34 +501,34 @@ class App extends Component {
 }
 ~~~~~~~~
 
-The sample data will represent data we will fetch from the API later on. An item in the list has a title, a url, and an author. Additionally, it comes with an identifier, points (which indicate how popular an article is), and a count of comments.
+The sample data represents information we will fetch from an API later on. Items in this list each have a title, a url, and an author, as well an identifier, points (which indicate how popular an article is), and a count of comments.
 
-Now you can use the built-in JavaScript `map` functionality in your JSX. It enables you to iterate over your list of items to display them. Again, you will use curly braces to encapsulate the JavaScript expression in your JSX.
+Now you can use the built-in JavaScript `map` functionality in JSX, which iterates over a list of items to display them according to a specified classification. Again, we use curly braces to encapsulate the JavaScript expression in JSX:
```

Comment:
is classification the correct word here?

Reply:
Yes, it is.  But I will change it to "according to specific attributes" since that sounds more code-related.